psyneulink/core/globals/utilities.py

```diff
@@ -1026,7 +1022,10 @@ def safe_create_np_array(value):
             # See https://numpy.org/neps/nep-0034-infer-dtype-is-object.html
             try:
                 try:
-                    return np.asarray(value)
+                    if torch and torch.is_tensor(value):
+                        return value
+                    else:
+                        return np.asarray(value)
```

Comment:
does this still need to be in a try block?

Reply:
Nope, I don't think so, moved it out.